ideascube/migrations/0009_add_a_system_user.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ideascube.models import User
+
+
+def add_user(*args):
+    User(serial='__system__', password='!!').save()
```

Comment:
Maybe add a full_name (like "System") so when we'll display the user in the front it will be more user friendly?